### DISCUSSION TITLE
Translate

### DISCUSSION THREAD

**Discussion by Thomas282-coder** [ScratchAddons/ScratchAddons#1068] [Translation] [answered]
Do you need to have the entire translation translated to publish, or are the approved translations published?

**Comment by WorldLanguages** [comment, marked as answer]
I'm not sure I understand, all I know is that it looks like we don't have any Polish translators yet

**Reply by mxmou** [reply]
I think @McGithub312 is asking if a language needs to be 100% translated for the translations to be added.

**Reply by WorldLanguages** [reply]
Oh. As of now, even if a language is 1% translated, that 1% goes in and users see the translated versions. But we could change that in the future.
